Manbil/Input/MovingCamera.h

```diff
@@ -26,17 +26,21 @@ class MovingCamera : public Camera
 
 
 	MovingCamera(float moveSpd = 10.0f, float rSpeed = 0.03f)
-        : pressedSpace(false), MoveSpeed(moveSpd), RotSpeed(rSpeed),
+        : pressedToggleKey(false), MoveSpeed(moveSpd), RotSpeed(rSpeed),
           mouseTarget(100, 100) { }
 	MovingCamera(Vector3f pos, float moveSpd = 10.0f, float rSpeed = 0.03f,
 				 Vector3f forward = Vector3f(1, 0, 0), Vector3f up = Vector3f(0, 0, 1),
                  bool lockUp = true)
-		: Camera(pos, forward, up, lockUp), pressedSpace(false),
+		: Camera(pos, forward, up, lockUp), pressedToggleKey(false),
           MoveSpeed(moveSpd), RotSpeed(rSpeed),
           mouseTarget(100, 100) { }
 
 
     bool IsMouseCapped(void) const { return capMouse; }
+	// turn mouse capture on or off
+	void SetMouseCapped(bool isCapped) {
+		capMouse = isCapped;
+	}
```

Comment:
Also also, the comment is unnecessary; it's just a recording of the actual function name.